testdata/go/v3/monitoring/memcached-operator/Makefile

```diff
@@ -271,3 +271,15 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+##@ Generate the metrics documentation
```

Comment:
@machadovilaca will we have documentation related to addition of this target in SDK docs as a follow up?

Reply:
Yes